CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
-## 0.7.1 (Unreleased)
+## 0.8.1 (Unreleased)
```

Comment:
This line needs to be actual version to be released next. 
The release tooling then timestamps it.